### PR TITLE
selenium: fox, chrome , selenide - chrome

### PR DESCRIPTION
**Pull request by VasilTsimashenka**
Screenshot_1
Screenshot_2
Screenshot_3

### CODE REVIEW

File: src/test/java/MyFirstATest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Test
+    public void openPage4() {
+        System.setProperty("selenide.browser", "firefox");
+        Selenide.open("http://automationpractice.com/index.php");
+    }
+
+    public static void main(String[] args) {
+        System.out.println();
+    }
```
Comment (Kirill-Simonovich) [inline src/test/java/MyFirstATest.java]: Delete psvm

File: src/test/java/SelenideFirefox.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import org.junit.jupiter.api.Test;
+
+public class SelenideFirefox {
+    @Test
+    public void openPage() {
+        System.setProperty("selenide.browser", "firefox");
+        Selenide.open("http://automationpractice.com/index.php");
+    }
+
+    public static void main(String[] args) {
```
Comment (Kirill-Simonovich) [inline src/test/java/SelenideFirefox.java]: Delete psvm

File: src/test/java/SelenideFirefox.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import com.codeborne.selenide.Selenide;
+import org.junit.jupiter.api.Test;
+
+public class SelenideFirefox {
```
Comment (Kirill-Simonovich) [inline src/test/java/SelenideFirefox.java]: Delete this class

File: src/test/java/MyFirstATest.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Test
+    public void openPage3() {
+        Selenide.open("http://automationpractice.com/index.php");
+    }
+
+    @Test
+    public void openPage4() {
+        System.setProperty("selenide.browser", "firefox");
```
Comment (Kirill-Simonovich) [inline src/test/java/MyFirstATest.java]: Please add screenshots of start Firefox browser by this test

**Review by Kirill-Simonovich — CHANGES_REQUESTED** [review]

**Comment by Kirill-Simonovich** [comment]
Please answer to comments on github

File: src/test/java/MyFirstATest.java (hunk 4 above)
Comment (Kirill-Simonovich) [reply src/test/java/MyFirstATest.java]: This is work?

File: src/test/java/SelenideFirefox.java (hunk 3 above)
Comment (Kirill-Simonovich) [reply src/test/java/SelenideFirefox.java]: What is this class for?

File: src/test/java/SelenideFirefox.java (hunk 2 above)
Comment (Kirill-Simonovich) [reply src/test/java/SelenideFirefox.java]: ?

**Review by Kirill-Simonovich — CHANGES_REQUESTED** [review]

**Comment by Kirill-Simonovich** [comment]
Add screenshots of success run

File: src/test/java/MyFirstATest.java (hunk 4 above)
Comment (VasilTsimashenka) [reply src/test/java/MyFirstATest.java]: Yes this is work, I sent screenshots)

Comment (VasilTsimashenka) [reply src/test/java/MyFirstATest.java]: Screenshot_2

Comment (VasilTsimashenka) [reply src/test/java/MyFirstATest.java]: Screenshot_1

Comment (VasilTsimashenka) [reply src/test/java/MyFirstATest.java]: Screenshot_3

**Review by Kirill-Simonovich — APPROVED** [review]

**Pull request merged**